lib/ash/policy/authorizer/authorizer.ex

```diff
@@ -830,12 +835,14 @@ defmodule Ash.Policy.Authorizer do
           Ash.Changeset.accessing(changeset, [:attributes, :calculations, :aggregates])
       end
 
+    pkey = Ash.Resource.Info.primary_key(query_or_changeset.resource)
+
     accessing_fields
+    # primary keys are always accessible
+    |> Enum.reject(&(&1 in pkey))
```

Comment:
As this was similar logic, I tried to make the same change here, but this led to tests failing. Before, if there were no field_policies at all, it would allow access, and now it behaves similarly to actions where you need to have a policy. 

How do you feel about this? Maybe something for 3.0?

Reply:
Also not sure how this plays together with the sort/filter when there are no policies at all.

Reply:
🤔 its a good question. We can consider it for 3.0, but will need to keep the current behavior until then.

Reply:
We should potentially have a top level check here for "is there at least one field policy" to make this decision explicit.